base/workflow/inst/batch_run.R

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env Rscript
-
-library(dplyr)
+library(tidyverse)
 library(PEcAn.workflow)
 stopifnot(
   requireNamespace("PEcAn.DB", quietly = TRUE),
   requireNamespace("PEcAn.utils", quietly = TRUE)
 )
-
+library(furrr)
```

Comment:
* @para2x: Consider moving this up so that all `library` calls are in one place.
* @ashiklom: I'm fuzzy on the best practice here: Does calling furrr in `inst` require adding it to Suggests?
...And while I'm looking at it, why the stopifnot for DB and utils above? They're both in workflow's Imports, so I'd be comfortable assuming they're available at call time.